### PR TITLE
Example of extending the session

### PR DESCRIPTION
**Pull request by bryce-gibson**
Adds an example of extending the session.

Hopefully this should answer #44 (happy to update if it doesn't help :-) )

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -148,6 +148,32 @@ app.use(function (req, res, next) {
...
+app.use(cookieSession({
+  name: 'session',
+  keys: ['key1', 'key2']
+}))
+
+// Update a value in the cookie so that the set-cookie will be sent.
+// Only changes every minute so that it's not sent with every request.
+app.use(function (req, res, next) {
+  req.session.nowInMinutes = Date.now() / 60e3
+})
```
Comment (codeaholics) [inline README.md]: Missing call to next()

Comment (bryce-gibson) [inline README.md]: Ah, whoops :-)

Thanks! :-)

**Comment by alvarotrigo** [comment]
+1

**Comment by Bluejay47** [comment]
Anyone have any thoughts on implementing some standard method to wrap this process? It seems like it would be easy to do. I don't mind giving it a try if anyone else thinks it might be a good idea.

**Comment by dougwilson** [comment]
Hi @Bluejay47, I would have added a `rolling` option to the module (to match that of `express-session`), but it completely falls apart because web browsers will make multiple parallel requests, and when one request modifies the session, the other requests will then overwrite it back with the old value, causing it to be almost impossible to actually make any changes to the session on a real web site (it will look like it works when using a very simple example, but doesn't work on an actual web site with many pages & assets).

**Pull request closed without merging**